.github/workflows/build-and-publish-image.yml

```diff
@@ -1,6 +1,7 @@
 name: Build and publish image
 
 on:
+  workflow_dispatch: {}
```

Comment:
```suggestion
```

This isn't useful here, we never run this workflow directly, it is included in other workflows, e.g. `testrunner.yml`.

Reply:
Aaah! That's why I couldn't find my own runs of it. Derp. Thank you. :D